lib/core/src/Cardano/Wallet.hs

```diff
@@ -293,6 +294,7 @@ data ErrCreateUnsignedTx
 data ErrEstimateTxFee
     = ErrEstimateTxFeeNoSuchWallet ErrNoSuchWallet
     | ErrEstimateTxFeeCoinSelection ErrCoinSelection
+    | ErrEstimateTxFeeBackend ErrValidateSelection
```

Comment:
The term `Backend` is a bit misleading here too. I'd rather stick with the naming convention and have something like:

```hs
ErrEstimateTxFeeValidateSelection ErrValidateSelection
```